virttest/env_process.py

```diff
@@ -690,9 +697,7 @@ def process(test, params, env, image_func, vm_func, vm_first=False):
     :param vm_first: Call vm_func first or not.
     """
     def _call_vm_func():
-        for vm_name in params.objects("vms"):
-            vm_params = params.object_params(vm_name)
-            vm_func(test, vm_params, env, vm_name)
+        return vm_func(test, params, env)
```

Comment:
But there is `process(test, params, env, preprocess_image, preprocess_vm)` and `preprocess_vm` has the last parameter `vm_name`

Reply:
Oh, ignore me, just see the `preprocess_vms` update.